src/components/DateRangePicker.jsx

```diff
@@ -198,6 +199,14 @@ export default class DateRangePicker extends React.Component {
     }
   }
 
+  getDateString(date) {
+    const { displayFormat } = this.props;
+    if (displayFormat) {
+      return date && date.format(displayFormat);
```

Comment:
when `date` is falsy, can you remind me what `toLocalizedDateString` will return?